src/main/java/buildcraft/core/Version.java

```diff
@@ -81,63 +78,7 @@ public static String getRecommendedVersion() {
 	}
 
 	public static void versionCheck() {
-		try {
```

Comment:
instead of deleting the corpse of the method, why don't you delete it totally? you deleted the call to it anyways, so no need to have a stub version check method

Reply:
it's a public method, and not sure where else it might be called

Reply:
is it shipped in the API? because i fail to see why would another mod relies on version check of buildcraft.

Reply:
fair enough, we'll find if it does heh